packages/playwright/src/generators/configuration/files/playwright.config.ts.template

```diff
@@ -1,5 +1,6 @@
 import { defineConfig } from '@playwright/test';
 import { nxE2EPreset } from '@nx/playwright/preset';
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
could we conditionally add this eslint ignore comment when the webServerCommand and webServerAddress aren't included?

```suggestion
<% if(!webServerCommand || !webServerAddress) { %>// eslint-disable-next-line @typescript-eslint/no-unused-vars <% } %>
```

Reply:
changed